docs/workforce-management-framework/topics/con-demo-app.adoc

```diff
@@ -2,5 +2,5 @@
 = Introducing the {WFM-RC-NameShort} Demo Application
 
 The {WFM-RC-NameShort} demo app comprises of two applications and are called *Mobile* and *Portal*.
-When the *Mobile* and *Portal* demo applications are running, they are accessed via a graphical user interface.
+When the *Mobile* and *Portal* demo applications are running, they are accessed on a mobile device.
```

Comment:
Totall missunderstanding. Please put this text:
`For testing and demo purposes mobile application and portal website and launched in the browser. Mobile application can be also build and installed on the mobile phone.`